nilearn/plotting/__init__.py

```diff
@@ -53,10 +53,11 @@ def _set_mpl_backend():
 
 __all__ = ['cm', 'plot_img', 'plot_anat', 'plot_epi',
            'plot_roi', 'plot_stat_map', 'plot_glass_brain',
-           'plot_connectome_strength', 'plot_connectome', 'plot_prob_atlas',
+           'plot_markers', 'plot_connectome', 'plot_prob_atlas',
```

Comment:
it looks like `plot_markers` is now included twice in this list ? You'll want to remove one instance